service-pdf/src/Opg/Lpa/Pdf/AbstractLp1.php

```diff
@@ -732,6 +871,66 @@ private function getTrustAttorney(array $attorneys)
         return null;
     }
 
+    private function writeBarcodeToImageFile()
```

Comment:
This is the main change which requires a review: changing the PDF rendering to use an image rather than a PDF. Once this is verified as working correctly, I will remove the code to render to a PDF. It's retained for reference until we can verify barcode image rendering works as intended.